modules/islandora_audio/js/audio.js

```diff
@@ -4,44 +4,87 @@
  * @file
  * Displays Audio viewer.
  */
-(function ($, Drupal) {
-    'use strict';
+(function (Drupal, once) {
+  Drupal.behaviors.customBehavior = {
+    attach: function (context, settings) {
 
-    /**
-     * If initialized.
-     * @type {boolean}
-     */
-    var initialized;
-    /**
-     * Unique HTML id.
-     * @type {string}
-     */
-    var base;
+      once('customBehavior', 'audio', context).forEach(function (element) {
```

Comment:
Same deal, should probably be namespaced in some way?

```suggestion
      once('islandora_audio_viewer', 'audio', context).forEach(function (element) {
```